src/inc/apiv2/helper/getCracksOfTask.routes.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+use DBA\Chunk;
+use DBA\ContainFilter;
+use JetBrains\PhpStorm\NoReturn;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use DBA\Factory;
+use DBA\Hash;
+use DBA\QueryFilter;
+use DBA\Task;
+use Middlewares\Utils\HttpErrorException;
+
+require_once(dirname(__FILE__) . "/../common/AbstractHelperAPI.class.php");
+
+class getCracksOfTaskHelper extends AbstractHelperAPI {
+  public static function getBaseUri(): string {
+    return "/api/v2/helper/getCracksOfTask";
+  }
+  
+  public static function getAvailableMethods(): array {
+    return ['GET'];
+  }
+  
+  public function getRequiredPermissions(string $method): array {
+    return [Hash::PERM_READ, Task::PERM_READ];
+  }
+  
+  public static function getResponse(): null {
+    return null;
+  }
+  
+  
+  #[NoReturn] public function actionPost(array $data): object|array|null {
+    assert(False, "getCracksOfTask has no POST");
+  }
+  
+  /**
+   * Description of get params for swagger.
+   */
+  public function getParamsSwagger(): array {
+    return [
+      [
+         "in" => "query",
+         "name" => "task",
+         "schema" => [
+           "type" => "integer",
+           "format" => "int32"
+         ],
+         "required" => true,
+         "example" => 1,
+         "description" => "The ID of the task."
+      ]
+    ];
+  }
+  
+  /**
+   * Endpoint to get the cracked hashes of a certain task
+   * @param Request $request
+   * @param Response $response
+   * @return Response
+   * @throws HttpErrorException
+   */
+  public function handleGet(Request $request, Response $response): Response {
+    $this->preCommon($request);
+    $task = Factory::getTaskFactory()->get($_GET['task']);
```

Comment:
The $_GET is probably a leftover from  the copying. This probably should be taken from the request.